src/OrleansRuntime/LogConsistency/ProtocolServices.cs

```diff
@@ -107,73 +121,65 @@ internal ProtocolServices(Grain gr, Logger log, IMultiClusterRegistrationStrateg
         /// <inheritdoc />
         public SerializationManager SerializationManager { get; }
 
-        public bool MultiClusterEnabled
-        {
-            get
-            {
-                return (PseudoMultiClusterConfiguration == null);
-            }
-        }
+        public bool MultiClusterEnabled => this.globalConfig.HasMultiClusterNetwork;
     
         public string MyClusterId
         {
             get
             {
-                return Silo.CurrentSilo.ClusterId;
+                return this.globalConfig.ClusterId;
             }
         }
 
+        private string ClusterDisplayName => this.pseudoMultiClusterConfiguration == null ? string.Empty : " " + this.MyClusterId;
```

Comment:
I think it's fine, although in many places you replaced `this.pseudoMultiClusterConfiguration == null` with `!this.MulticlusterEnabled`